docs/proposals/DataTable/README.md

```diff
@@ -0,0 +1,165 @@
+# DataTable
+
+The `DataTable` Component would give the users options for pagination, sorting,
+and column visibility by leveraging some sort of API or service.
+
+## Design Patterns
+
+Tables are used to organize tabular information to help users understand and
+quickly scan for information.
+
+Some scenarios for Tables include the following:
+
+- Index Pages
+- Dashboards
+
+### Sorting
+
+Sorting can be enabled on column headers to improve usability.
+
+The default sorted column should be based on contextual based on user needs.
+
+To indicate the column being sorted, use an up (ascending) or down (descending)
+arrow next to the column title. Only 1 column can be sorted at a time. User can
```

Comment:
> Only 1 column can be sorted at a time

Is this a package limitation or just a limitation we are imposing here?

Reply:
It would be a limitation we would be imposing at this moment. The library has the capability to do [multi-sorting](https://tanstack.com/table/v8/docs/api/features/sorting#enablemultisort).

Reply:
Okay, I assume you're waiting on that because it adds complexity for something not being asked for yet?
Might be worth documenting that it is a limitation we are imposing, in case someone needs to build this in the future